### PR TITLE
Eng 19294 release topic gaps

### PR DESCRIPTION
**Pull request by rdykiel**

### CODE REVIEW

File: src/frontend/org/voltdb/SystemProcedureCatalog.java [hunk 1]
```diff
@@ -632,6 +633,12 @@ Procedure asCatalogProcedure() {
                         false, false, false, 0, VoltType.INVALID,
                         false, false, true, Durability.NOT_DURABLE,
                         false, true, Restartability.RESTARTABLE));
+        // @TopicControl is like @ExportControl but for topic streams in PRO
+        builder.put("@TopicControl",
```
Comment (n3world) [inline src/frontend/org/voltdb/SystemProcedureCatalog.java:637]: I would recommend using the build I find it a little easier to read than the giant constructor

File: src/frontend/org/voltdb/sysprocs/TopicControl.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            TopicControlOperation.valueOf(operationMode.toUpperCase());
+        } catch (IllegalArgumentException e){
+            results.addRow("", -1, "FAILURE", e.getMessage());
+            return new VoltTable[] {results};
+        }
+
+        topic = topic == null ? "" : topic;
+        if (!StringUtil.isEmpty(topic)) {
+            if (!CatalogUtil.isTopic(ctx.getDatabase(), topic)) {
+                results.addRow(topic, -1,"FAILED", "Topic stream " + topic + " does not exist.");
```
Comment (n3world) [inline src/frontend/org/voltdb/sysprocs/TopicControl.java:105]: The word stream in this error message seams strange

File: src/frontend/org/voltdb/SystemProcedureCatalog.java (hunk 1 above)
Comment (rdykiel) [reply src/frontend/org/voltdb/SystemProcedureCatalog.java:637]: No changes. Keeping in line with surrounding code

File: src/frontend/org/voltdb/sysprocs/TopicControl.java [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+import org.voltdb.VoltTable.ColumnInfo;
+import org.voltdb.VoltType;
+import org.voltdb.export.ExportManagerInterface;
+import org.voltdb.utils.CatalogUtil;
+import org.voltdb.utils.VoltTableUtil;
+
+/**
+ * Release a topic that may be in BLOCKED state
+ */
+public class TopicControl extends VoltSystemProcedure {
```
Comment (n3world) [inline src/frontend/org/voltdb/sysprocs/TopicControl.java:39]: This should actually be in pro since topics are a pro feature

**Pull request merged**